macos/FluentUI/FilledTemplateImageView/FilledTemplateImageView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+/// Custom view for displaying a template style image that can be filled with a specified color. This takes in an image with a corresponding
+/// overlay mask that is used for drawing the fill. The image and its corresponding fillMask need to line up appropriately so that when
+/// the image is drawn over its mask, its outline will perfectly overlap the edges of the mask to achieve the desired fill effect.
+@objc(MSFFilledTemplateImageView)
+open class FilledTemplateImageView: NSImageView {
+
+	/// Creates a custom template style image view with a fill.
+	/// - Parameters:
+	///   - image: the template style image or icon to be drawn.
+	///   - fillMask: the mask image used to draw the fill color.
+	///   - contentTintColor: the color to use for the main image outline.
+	///   - fillColor: the color to use for the image fill. When set to nil or clear, the fill won't be drawn.
+	@objc(initWithImage:fillMask:contentTintColor:FillColor:)
```

Comment:
Sorry, just noticed this. Two questions:
- Why are you declaring the Objective-C method signature explicitly? Is this not what it would be generated as if you just say `@objc`? I'm not 100% myself, TBH.
- If we are sticking with this, `FillColor` should be capitalized as `fillColor`.

Reply:
Oops, good catch on accidentally capitolizing "FillColor". Regarding the @objc(...) line, we typically leave these for better objc naming. That said, if I remove that line it actually doesn't declare any special initializer in the objc header (I figured I'd at least get some morphed Swift version of it), so definitely needed in this case.

Reply:
Did you also remove `@objc`? Should be:
```
@objc public init(with
		image: NSImage,
		fillMask: NSImage,
		contentTintColor: NSColor,
		fillColor: NSColor?
```

Reply:
But yeah, I'll add the with decorator for Swift.

Reply:
Just saw this. Yeah, I did remove @objc! : )